Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionAutomatedALMDefects.cs

```diff
@@ -134,8 +134,8 @@ public override void Execute(ReportInfo RI)
                 else
                     return;
                 var defectFields = defaultALMDefectProfile.ALMDefectProfileFields.Where(a => a.Mandatory || a.ToUpdate).ToList();
-
-                RepositoryItemHelper.RepositoryItemFactory.CreateNewALMDefects(defectsForOpening, defectFields);
+                //update alm type to open defect
+                RepositoryItemHelper.RepositoryItemFactory.CreateNewALMDefects(defectsForOpening, defectFields, defaultALMDefectProfile.AlmType);
```

Comment:
If user will have 2 runset operations

1. Open ALM Defect Jira
2. Publish the results to qTest or QC

The code will work ? please test this scenario

Reply:
Failed. Did changes accordingly